modules/core/src/main/scala/org/tessellation/modules/Storages.scala

```diff
@@ -25,10 +27,12 @@ object Storages {
   def make[F[_]: Async: KryoSerializer: Supervisor](
     sdkStorages: SdkStorages[F],
     snapshotConfig: SnapshotConfig,
-    maybeRollbackHash: Option[Hash]
+    maybeRollbackHash: Option[Hash],
+    trustUpdates: Option[InternalTrustUpdateBatch]
   ): F[Storages[F]] =
     for {
       trustStorage <- TrustStorage.make[F]
+      _ <- trustUpdates.fold(Applicative[F].pure(()))(trustStorage.updateTrust(_))
```

Comment:
There is an Applicative unit, so you dont need to write pure(())

Reply:
Changed.